### ISSUE TITLE
Bug: Call to undefined method CodeIgniter\HTTP\CLIRequest::getPost() while running the CLI routes

### ISSUE THREAD

**Issue by DhPandya** [codeigniter4/CodeIgniter4#6419]
### PHP Version

8.0

### CodeIgniter4 Version

4.2.4

### CodeIgniter4 Installation Method

Composer (using `codeigniter4/appstarter`)

### Which operating systems have you tested for this bug?

Windows

### Which server did you use?

apache

### Database

MySQL 8.0.27

### What happened?

Running CLI routes is getting an error
![image](https://user-images.githubusercontent.com/46279781/186424242-3c5f8df3-8b12-4e4d-92fa-1d10bff8975b.png)

### Steps to Reproduce

1. Declare any CLI routes 
2. Try to execute it using CMD/Terminal

### Expected Output

It should execute the CLI request instead of the error

**Comment by datamweb** [comment]
Duplicate of: #6365 , #6409

**Comment by kenjis** [comment]
I have confirmed this bug.

```diff
--- a/app/Config/Filters.php
+++ b/app/Config/Filters.php
@@ -33,13 +33,13 @@ class Filters extends BaseConfig
      */
     public $globals = [
         'before' => [
-            // 'honeypot',
+             'honeypot',
             // 'csrf',
             // 'invalidchars',
         ],
         'after' => [
             'toolbar',
-            // 'honeypot',
+             'honeypot',
             // 'secureheaders',
         ],
     ];
```

```console
$ php public/index.php 


[Error]

Call to undefined method CodeIgniter\HTTP\CLIRequest::getPost()

at SYSTEMPATH/Honeypot/Honeypot.php:62

Backtrace:
  1    SYSTEMPATH/Filters/Honeypot.php:34
       CodeIgniter\Honeypot\Honeypot()->hasContent(Object(CodeIgniter\HTTP\CLIRequest))

  2    SYSTEMPATH/Filters/Filters.php:173
       CodeIgniter\Filters\Honeypot()->before(Object(CodeIgniter\HTTP\CLIRequest), null)

  3    SYSTEMPATH/CodeIgniter.php:431
       CodeIgniter\Filters\Filters()->run('', 'before')

  4    SYSTEMPATH/CodeIgniter.php:340
       CodeIgniter\CodeIgniter()->handleRequest(null, Object(Config\Cache), false)

  5    FCPATH/index.php:67
       CodeIgniter\CodeIgniter()->run()
```

**Comment by kenjis** [comment]
I sent a PR to fix: #6421
Please try it if you can.

**Comment by DhPandya** [comment]
> I sent a PR to fix: #6421 Please try it if you can.

@kenjis Thanks for the update & Fix.
I've tried the fix you mentioned and it is working fine by updating the files manually.
